### ISSUE TITLE
Provide multiple JUnit 5 Test Suite to address the multiple deployment issue.

### ISSUE THREAD

**Issue by charleech** [charleech/arquillian-junit5-payara#5]
Provide multiple JUnit 5 Test Suite to address the multiple deployments.

If there are more than one test suite by using `@RunWith(JUnitPlatform.class)`, only the first executed test suite will be success and all the rest are failed with the following message: -

```
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 38.205 s - in xxxTestSuite
[INFO] Running yyyTestSuite
[ERROR] Tests run: 4, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 0.144 s <<< FAILURE! - in yyyTestSuite
[ERROR] DependencyLoaderIntgrtnTester  Time elapsed: 0.1 s  <<< ERROR!
java.lang.RuntimeException: Could not setup GlassFish Embedded Bootstrap
Caused by: org.glassfish.embeddable.GlassFishException: Already bootstrapped

[ERROR] JUnit Jupiter  Time elapsed: 0.11 s  <<< ERROR!
java.lang.RuntimeException: Could not run @AfterSuite
Caused by: java.lang.IllegalStateException: There is no Graphene instance at the moment
```

**Comment by charleech** [comment]
The current message with this project by using the following command is as the following: -

```bash
mvn clean install -Dpayara-embedded=true
```


```
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running it.test.app.github.charleech.my.cdi.MyCdiJunit5IntgrtnTestSuite
2021-05-08 10:05:32.186 [main] [] INFO  org.reflections.Reflections                   - 232 scan Reflections took 67 ms to scan 1 urls, producing 7 keys and 16 values 
2021-05-08 10:06:01.119 [http-thread-pool::https-listener(1)] [] INFO  app.github.charleech.my.cdi.Greeter           - 31 greet The parameter=JUnit5_1
2021-05-08 10:06:01.773 [http-thread-pool::https-listener(4)] [] INFO  app.github.charleech.my.cdi.Greeter           - 31 greet The parameter=JUnit5_2
JdbcRuntimeExtension,  getAllSystemRAResourcesAndPools = [GlassFishConfigBean.org.glassfish.jdbc.config.JdbcResource, GlassFishConfigBean.org.glassfish.jdbc.config.JdbcResource, GlassFishConfigBean.org.glassfish.jdbc.config.JdbcConnectionPool, GlassFishConfigBean.org.glassfish.jdbc.config.JdbcConnectionPool]
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 33.533 s - in it.test.app.github.charleech.my.cdi.MyCdiJunit5IntgrtnTestSuite
[INFO] Running it.test.app.github.charleech.my.cdi.MyCdiJunit5Round2IntgrtnTestSuite
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.017 s <<< FAILURE! - in it.test.app.github.charleech.my.cdi.MyCdiJunit5Round2IntgrtnTestSuite
[ERROR] MyCdiJunit5IntgrtnTester  Time elapsed: 0.016 s  <<< ERROR!
java.lang.RuntimeException: Could not setup GlassFish Embedded Bootstrap
Caused by: org.glassfish.embeddable.GlassFishException: Already bootstrapped

[ERROR] Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF-8
[ERROR] OpenJDK 64-Bit Server VM warning: ignoring option MaxPermSize=256m; support was removed in 8.0
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   Could not setup GlassFish Embedded Bootstrap
[INFO] 
[ERROR] Tests run: 3, Failures: 0, Errors: 1, Skipped: 0
```

**Comment by charleech** [comment]
Closed by  #6
